apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -581,15 +573,41 @@ int ble_rmc_main(int argc, char *argv[])
 					RMC_LOG(RMC_CLIENT_TAG, "Fail to set timer\n");
 				} else {
 					scan_time = (uint32_t)temp;
-					RMC_LOG(RMC_CLIENT_TAG, "Timer : %u s\n", scan_time);
 				}
 			}
+			RMC_LOG(RMC_CLIENT_TAG, "Timer : %us\n", scan_time);
 
 			ble_scan_filter filter = { 0, };
 			filter.raw_data_length = 0;
 			filter.scan_duration = scan_time * 1000; // scan_duration should be in milliseconds.
+			filter.whitelist_enable = true;
+			scan_config.device_scanned_cb = ble_device_scanned_cb_for_test;
+			ret = ble_client_start_scan(&filter, &scan_config);
+
+			if (ret != BLE_MANAGER_SUCCESS) {
+				RMC_LOG(RMC_CLIENT_TAG, "scan start fail[%d]\n", ret);
+				goto ble_rmc_done;
+			}
+		} else if (argc >= 3 && argv[2][0] == '3') {
```

Comment:
with this code argument 30 3 300 execute same code. Is it acceptable?

Reply:
I will fix it to use 'strncmp'